plugins/woocommerce/includes/react-admin/wc-admin-update-functions.php

```diff
@@ -45,6 +45,12 @@ function wc_admin_update_0201_order_status_index() {
 function wc_admin_update_0230_rename_gross_total() {
 	global $wpdb;
 
+	// The column might not exist, since this update function was moved from 230 to 400
+	// E.g. when updating from 250 to 800
+	if ( ! $wpdb->query( "SHOW COLUMNS FROM {$wpdb->prefix}wc_order_stats LIKE 'gross_total'" ) ) {
```

Comment:
I think it should get_results instead, I don't think `$wpdb->query` return value behavior is defined for `SHOW COLUMNS` query, and as such can change in future.
```suggestion
	if ( empty( $wpdb->get_results( "SHOW COLUMNS FROM {$wpdb->prefix}wc_order_stats LIKE 'gross_total'" ) ) ) {
```

Reply:
No, `query` is for the execution of arbitrary commands and is in fact the correct way to do this. If you check `get_results` will actually call `query` under the hood.